### ISSUE TITLE
Support receive function

### ISSUE THREAD

**Issue by chrsengel** [ConsenSys/surya#115]
Currently it doesn't support the `receive` function introduced in Solidity 0.6.0 as described [here](https://solidity.readthedocs.io/en/v0.6.0/contracts.html#receive-ether-function).

```
/home/chris/.npm-global/lib/node_modules/surya/node_modules/yargs/yargs.js:1109
      else throw err
           ^
ParserError: mismatched input '(' expecting {'from', 'calldata', 'constant', 'internal', 'private', 'public', Identifier} (79:9)
...
```

**Comment by chrsengel** [comment]
Guess this issue could be called "Support Solidity 0.6.0" as there are some more issues concerning this version. E.g. the missing `fallback` function.

**Comment by GNSPS** [comment]
Hey @chrsengel ! Thank you for reporting this! I'm taking on support for `v0.6.0` and hopefully have it completed really soon!

**Comment by GNSPS** [comment]
This was fixed in #116 ! 🎉 

A major update to the parser that we are maintaining ourselves now! 🚀 Closing.
